docs/study-def-dates.md

```diff
@@ -40,7 +41,7 @@ study = StudyDefinition(
 )
 ```
 
-This can make it easier to change the index date of a study by making sure it is only defined in one place.
+This can make it easier to change the index date of a study by making sure it is only defined in one place. (Note that if using `between`, ensure that the dates are given in chronological order).
```

Comment:
add example? e.g., `between = "index_date - 1 month", "index_date`

Reply:
I've referred to the example that is directly above since it is already there, does that look ok?

Reply:
Looks good